test/compress/object.js

```diff
@@ -22,7 +46,31 @@ getter_setter: {
         var get = "bar";
         var a = {
             get,
-            set: "foo"
+            set: "foo",
+            get bar() {
+                return this.get;
+            },
+            get 5() {
+                return "five";
+            },
+            get 0x55() {
+                return "five five";
+            },
+            get "five"() {
```

Comment:
If it works in node, it generally works in Chrome - both use v8 engine. Node v8 usually lags the browser version.